_articles/best-practices.md

```diff
@@ -82,6 +82,8 @@ If you meet with other maintainers, or make a major decision in private, documen
 
 That way, anybody who joins your community will have access to the same information as someone who's been there for years.
 
+As a open source maintainer you would like being respectful and encouraging for open source contributors. People contributing in an open source project is a gain for everyone stakeholder involved, be you as a maintainer or the one is contributing. For helping you in the endeavour we have good first issues, if you label issues as good first issues we will show the issues as a GitHub feature.
```

Comment:
I'd drop this paragraph. The rest of this section is higher level and I don't think good first issues is better in the building community article.

Reply:
I'm not sure this is the correct place for this. I suggest adding a paragraph in the section `Share the workload`, right after the sentence "If you're looking for others to pitch in, start by asking around." The new paragraph could be something like this:

"One way to gain new contributors is to explicitly [label issues that are simple enough for beginners to tackle](https://help.github.com/en/articles/helping-new-contributors-find-your-project-with-labels). GitHub will then surface these issues in various places on the platform, increasing their visibility."

Reply:
0ffac237dab08e572552f11fc7eeb3d717913172.